internal/internal_worker_base.go

```diff
@@ -141,7 +142,7 @@ type (
 		logger               *zap.Logger
 		metricsScope         tally.Scope
 
-		pollerRequestCh    chan struct{}
+		dynamic            *worker.DynamicParams
```

Comment:
see my other comment on renaming. I'd also rename this member variable to `concurrencyLimits`. `dynamic` is too generic

Reply:
changed